nav2_behavior_tree/plugins/action/back_up_action.cpp

```diff
@@ -31,10 +31,9 @@ BackUpAction::BackUpAction(
   double speed;
   getInput("backup_speed", speed);
 
-  // silently fix, vector direction determined by distance sign
-  if (speed < 0.0) {
-    speed *= -1.0;
-  }
+  // Silently ensure, that speed and distance are positive values
+  speed = std::fabs(speed);
```

Comment:
This is not necessary if we do i in the backup recovery itself. We can remove these 3 lines in favor for the 2 you added in back_up.cpp